it/repositories/ActiveSessionRepositorySpec.scala

```diff
@@ -17,32 +17,36 @@
 package repositories
 
 import models.UtrSession
-import org.scalatest._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration
+import org.mongodb.scala.bson.BsonDocument
+import org.scalatest.{BeforeAndAfterEach, OptionValues}
+import uk.gov.hmrc.mongo.test.MongoSupport
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.matchers.must.Matchers
-import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.wordspec.AnyWordSpec
+import scala.concurrent.Await
 
-class ActiveSessionRepositorySpec extends AnyFreeSpec with Matchers
-  with ScalaFutures with OptionValues with MongoSuite {
+class ActiveSessionRepositorySpec extends AnyWordSpec with Matchers
+  with ScalaFutures with OptionValues with MongoSupport with MongoSuite with BeforeAndAfterEach {
 
-  "a session repository" - {
+  override def beforeEach() = Await.result(repository.collection.deleteMany(BsonDocument()).toFuture(),Duration.Inf)
```

Comment:
`await(repository.collection.deleteMany(BsonDocument()).toFuture())`

Could just swap out for this and `import play.api.test.Helpers.{await, defaultAwaitTimeout}`

Reply:
done